ghost/ghost/src/db/knex/activity.repository.knex.ts

```diff
@@ -0,0 +1,53 @@
+import {Inject} from '@nestjs/common';
+import ObjectID from 'bson-objectid';
+import {ActivityRepository} from '../../core/activitypub/activity.repository';
+import {Activity} from '../../core/activitypub/activity.entity';
+
+export class ActivityRepositoryKnex implements ActivityRepository {
+    constructor(
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        @Inject('knex') private readonly knex: any
+    ) {}
+
+    #modelToActivity(model: any) {
```

Comment:
@allouis I don't know if that's needed, but I expect that the repository has to return an `Actor` type, so not sure how else to serialise it...

Reply:
The Repository deffo has to return an `Activity` type - this should be giving TS errors 🤔 

You should be able to `return new Activity(data)` here, or if you need a factory, create a new factory method and do `return Activity.create(data)`